pages/api/interest-zones/index.ts

```diff
@@ -0,0 +1,20 @@
+import ZoneSchema from '../../../data/zone.schema';
+import { dbConnection } from '../../../services/data.service';
+
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+const databaseConnection = dbConnection;
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse<any>) {
+    switch (req.method) {
+        case 'GET':
+            try{
+                const interestZoneList = await ZoneSchema.find();
```

Comment:
Please move the above code to a dedicated service as well. It is only one line of code, however it can be a best practice to always use an abstraction layer between the HTTP controller and the database call.